services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/models/resource_tracker_container.py

```diff
@@ -60,8 +68,8 @@ class ContainerScrapedResourceUsage(
 
 
 class ContainerGetDB(BaseModel):
-    service_settings_reservation_nano_cpus: int | None
-    service_settings_reservation_memory_bytes: int | None
+    cpu_limit: float
+    memory_limit: int
```

Comment:
see above: or `null`?

Reply:
for now, I would like to keep it not nullable -> these numbers are the ones that the user specifies from the front-end and we will need to use them in the pricing (so I want it to fail quickly if there is some problem, but if we will need the null value in the future we will add it)